guides/common/modules/ref_redhat-cloud-settings.adoc

```diff
@@ -8,8 +8,5 @@
 | *Synchronize recommendations Automatically* | No | Enable automatic synchronization of Insights recommendations from the {Team} cloud.
 | *Obfuscate host names* | No | Obfuscate hostnames sent to the {Team} cloud.
 | *Obfuscate host ipv4 addresses* | No | Obfuscate IPv4 addresses sent to the {Team} cloud.
-| *{Team} Cloud token* | \\***** | Authentication token sent to the {Team} cloud services.
-Used to authenticate requests to cloud APIs.
-| *Exclude installed Packages* | No | Exclude installed packages from upload to the {Team} cloud.
-| *Include parameters in insights-client reports* | No | Include parameter tags from {Project} during import.
+| *ID of the {RHCShort}(Yggdrasil) daemon* | \\***** | {RHCShort} daemon id.
```

Comment:
```suggestion
| *ID of the {RHCShort} daemon* | \\***** | {RHCShort} daemon id.
```

Reply:
That's what also confused me, but it's literally there in the rh_cloud sources. See https://github.com/theforeman/foreman-documentation/pull/2180#discussion_r1187735551

Reply:
Agree with Ewoud. It's visible.

Reply:
I think in older versions the name Yggdrasil was used as the package name.

Reply:
@ehelms are you arguing that foreman_rh_cloud's description should be changed?

Reply:
> @ehelms are you arguing that foreman_rh_cloud's description should be changed?

Sorry, which description explicitly?

Reply:
Yes. I see this on WebUI. :+1:

Reply:
Hi @ehelms @ekohl 
Are we good to move forward since `Yggdrasil` is visible?